wfe/context.go

```diff
@@ -0,0 +1,85 @@
+package wfe
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/jmhodges/clock"
+	"github.com/letsencrypt/boulder/core"
+	blog "github.com/letsencrypt/boulder/log"
+)
+
+type requestEvent struct {
+	ID           string    `json:",omitempty"`
+	RealIP       string    `json:",omitempty"`
```

Comment:
Oh, I see, this comes from `X-Real-IP`, which appears to be how the client's real IP gets passed from the front end to the back end.  Given that, I can live with the name, but in general let's put comments on these fields, since some of them are non-obvious. In this case, something like

```
// The end client's IP address, from X-Real-IP
```

(Yes, I see that these fields are copied from elsewhere, but let's take the opportunity to clean up.)